print-bgp.c

```diff
@@ -2698,7 +2699,7 @@ bgp_notification_print(netdissect_options *ndo,
 		    else {
 			    ND_TCHECK2(*(tptr+1), shutdown_comm_length);
 			    ND_PRINT((ndo, ", Shutdown Communication (length: %u): \"", shutdown_comm_length));
-			    safeputs(ndo, tptr+1, shutdown_comm_length);
+			    safeputlchars(ndo, tptr+1, shutdown_comm_length);
```

Comment:
No need for a new print function, you should use `fn_printn(ndo, tptr+1, shutdown_comm_length, NULL);`